.env.local

```diff
@@ -5,4 +5,6 @@ REACT_APP_BLOCK_URLS=https://polygonscan.com
 REACT_APP_CHAIN_NAME=Matic
 ESLINT_NO_DEV_ERRORS=true
 REACT_APP_API_NODE_URL=https://polygon-rpc.com/
-REACT_APP_MULTICALL_CONTRACT_ADDRESS=0x11ce4B23bD875D7F5C6a31084f55fDe1e9A87507 
\ No newline at end of file
+REACT_APP_MULTICALL_CONTRACT_ADDRESS=0x11ce4B23bD875D7F5C6a31084f55fDe1e9A87507 
+REACT_APP_SENTRY_DSN=
+REACT_APP_SENTRY_ENVIRONMENT=development
```

Comment:
Does this require I set these in the deployment environment?

Reply:
Yes, we need set envs on all environments, but keep them with same values.

Create a new sentry account is needed.

Reply:
@mikeghen You can [find the DSN here](https://docs.sentry.io/product/sentry-basics/dsn-explainer/#where-to-find-your-dsn).

Reply:
And Sentry will not college anything in `development` environment, because the `DSN` is not provided.

see more details here: [Where the DSN does](https://docs.sentry.io/product/sentry-basics/dsn-explainer/#what-the-dsn-does)

Reply:
Got it, let me take a look at getting setup with sentry